### PR TITLE
#1103592 add support for sql server

### PR DESCRIPTION
**Pull request by RaphaelAudet**
Signed-off-by: Raphael Audet <raphael.audet@mendix.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -38,7 +38,7 @@ RUN mkdir -p "/root/.java/.userPrefs/com/mendix/core"
 RUN ln -s "/.java/.userPrefs/com/mendix/core/prefs.xml" "/root/.java/.userPrefs/com/mendix/core/prefs.xml"
 
 # Start up application
-COPY scripts /build
+COPY scripts/ /build
```
Comment (carlos-salinas) [inline Dockerfile]: Please specify the startup script as you did in the compilations phase, e.g. :

```COPY scripts/startup /build```

File: Makefile [hunk 2]
```diff
@@ -8,5 +8,9 @@ build-image:
 	docker build \
 	--build-arg BUILD_PATH=build \
 	-t mendix/mendix-buildpack:v1 .
-run-container:
-	docker-compose up
+test-container:
+	tests/test-generic.sh tests/docker-compose-postgres.yml
+	tests/test-generic.sh tests/docker-compose-sqlserver.yml
+	tests/test-generic.sh tests/docker-compose-azuresql.yml
```
Comment (carlos-salinas) [inline Makefile]: Can you please document the behaviour of the SQL Server and Azure SQL Server databases and why you had to create a custom image for them.

**Comment by RaphaelAudet** [comment]
@carlos-salinas Tests are ok, could you merge ? (i've added docs updates)

**Comment by carlos-salinas** [comment]
LGTM

**Pull request merged**